packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -256,16 +258,17 @@ The functions above are often used in tandem to check whether a user has an AD o
-| Refer to a `Number` constant                                                                | `3.141`                                     |
-| Refer to a `Boolean` constant                                                               | `true`                                      |
-| Refer to an `Array` element                                                                 | `{1, 2, 3}[0]` or `user.arrayProperty[0]`   |
-| Concatenate two strings                                                                     | `user.firstName + user.lastName`            |
-| Concatenate two strings with space                                                          | `user.firstName + " " + user.lastName`      |
-| Ternary operator example:<br>If group code is 123, assign value of Sales, else assign Other | `user.groupCode == 123 ? 'Sales' : 'Other'` |
+| Common Actions                                                                                                                    | Example                                         |
+| ----------------                                                                                                                  | --------                                        |
+| Refer to a `String` constant                                                                                                      | `'Hello world'`                                 |
+| Refer to an `Integer` constant                                                                                                    | `1234`                                          |
+| Refer to a `Number` constant                                                                                                      | `3.141`                                         |
+| Refer to a `Boolean` constant                                                                                                     | `true`                                          |
+| Refer to an `Array` element                                                                                                       | `{1, 2, 3}[0]` or `user.arrayProperty[0]`       |
+| Concatenate two strings                                                                                                           | `user.firstName + user.lastName`                |
+| Concatenate two strings with space                                                                                                | `user.firstName + " " + user.lastName`          |
+| Ternary operator example:<br>If group code is 123, assign value of Sales, else assign Other                                       | `user.groupCode == 123 ? 'Sales' : 'Other'`     |
+| Elvis operator<br>If `Groups.startsWith("OKTA", "TEST", 100)` is non-NULL, return that value, otherwise return `{}` (empty list)  | `Groups.startsWith("OKTA", "TEST", 100) ?: {}`  |
```

Comment:
Hmm I'm not sure what we should call this operator? I can see [there are few similar ones](https://github.com/okta/okta-developer-docs/blob/6bcee6ca27b59e19de21df2004940cf41818fbc8/packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md?plain=1#L178): `contains`, `startsWith` and `endWiths`, they would try to match the `pattern`, returns the matched group names in a string or a list, or return null or error if it doesn't match 🤔

Reply:
Hmm it feels these functions are not replying on SpEL's Elvis, and they work differently from the Elvis operator too, because you can't design the value to be returned when there is no match, it would always return null.